packages/contracts-bedrock/test/L1/LiquidityMigrator.t.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import { Test } from "forge-std/Test.sol";
+import { LiquidityMigrator } from "src/L1/LiquidityMigrator.sol";
+import { SharedLockbox } from "src/L1/SharedLockbox.sol";
+
+contract LiquidityMigratorTest is Test {
```

Comment:
Maybe when having the `SharedLockbox` in the `Setup` we can also use `CommonTest` here